src/modules/RootPart.rb

```diff
@@ -1488,12 +1340,6 @@ def MountPartitions(root_device_current)
 
               success = false
             else
-              Builtins.y2milestone("cryptotab %1", crtab)
-
-              fstab_ref = arg_ref(fstab)
-              PrepareCryptoTab(crtab, fstab_ref)
-              fstab = fstab_ref.value
-
               Builtins.y2milestone("fstab %1", fstab)
```

Comment:
does this log still makes sense?

Reply:
Maybe yes. At this point the fstab is kind of valid and the milestone is achieved. Yes, its value does not change because `PrepareCryptoTab` was removed, but I would still log it. In fact, this is the only one point where fstab is logged.